scripts/install.sh

```diff
@@ -277,6 +271,18 @@ while getopts ":h-:" optchar; do
   esac
 done
 
+if [ ! -d "$MODULEDIR/helm-charts" ]; then
+  if  [ ! -d "$SCRIPTDIR/helm-charts" ]; then
+    git clone --quiet -c advice.detachedHead=false -b $HELMCHARTVERSION https://github.com/dell/helm-charts
+  fi
+  mv helm-charts $MODULEDIR
+else
+  if [  -d "$SCRIPTDIR/helm-charts" ]; then
+    rm -rf $SCRIPTDIR/helm-charts
```

Comment:
Any reason we are not removing $MODULEDIR?

Reply:
To make sure that helm-charts exists only at MODULEDIR and not at SCRIPTDIR.